src/main/java/io/conduit/SourceService.java

```diff
@@ -86,31 +88,44 @@ public void start(Source.Start.Request request, StreamObserver<Source.Start.Resp
         } catch (Exception e) {
             Logger.get().error("Error while starting.", e);
             responseObserver.onError(
-                    Status.INTERNAL.withDescription("couldn't start task: " + e.getMessage()).withCause(e).asException()
+                Status.INTERNAL.withDescription("couldn't start task: " + e.getMessage())
+                    .withCause(e)
+                    .asException()
             );
         }
     }
 
     @Override
     public StreamObserver<Source.Run.Request> run(StreamObserver<Source.Run.Response> responseObserver) {
         this.runStream = new DefaultSourceStream(
-                task,
-                position,
-                responseObserver,
-                Transformations::fromKafkaSource
+            task,
+            position,
+            responseObserver,
+            getTransformation()
         );
         runStream.startAsync();
         return runStream;
     }
 
+    private Function<SourceRecord, Record.Builder> getTransformation() {
```

Comment:
Another renaming proposal - `getRecordConverter` or something with "converter" in it. I don't want us to confuse this with Conduit processors since we occasionally call them transforms.

Reply:
That makes sense. :+1: